app/server/lib/sendAppPage.ts

```diff
@@ -158,9 +164,9 @@ export function makeSendAppPage({ server, staticDir, tag, testLogin, baseDomain
     const tagManagerSnippet = needTagManager ? getTagManagerSnippet(process.env.GOOGLE_TAG_MANAGER_ID) : '';
     const staticTag = options.tag || tag;
     // If boot tag is used, serve assets locally, otherwise respect
-    // APP_STATIC_URL.
-    const staticOrigin = staticTag === 'boot' ? '' : (process.env.APP_STATIC_URL || '');
-    const staticBaseUrl = `${staticOrigin}/v/${staticTag}/`;
+    // APP_STATIC_URL or APP_HOME_URL.
+    const staticOrigin = staticTag === 'boot' ? '' : (process.env.APP_STATIC_URL || config.homeUrl || '');
+    const staticBaseUrl = `${trimEnd(staticOrigin, '/')}/v/${staticTag}/`;
```

Comment:
Another scary change. The motivation was `as I now specify the absolute URL for serving static resources (needed for specifying the opengraph icon)`? Do you actually still need this?

Reply:
I removed the changes that looked scary to you, with the last changes it was indeed not worth the risk, you're right.